docs/utils/colors.js

```diff
@@ -65,4 +65,7 @@ export const teal = [
 ];
 export const dodgerblue = [
   {'name':'dodger-blue', 'hax':'#A3BAE0', 'rgb':''},
+];
+export const terminalblue = [
+  {'name':'terminal-blue', 'hax':'#3D51AD', 'rgb':''},
```

Comment:
Add this color at the beginning of this file in the comments

Reply:
- Added